scripts/generator/graphql_generator/csharp/SDK/INativePayable.cs.erb

```diff
@@ -0,0 +1,10 @@
+namespace <%= namespace %>.SDK {
+
+    interface INativePayable {
+        bool CanShowPaymentSetup();
+        void ShowPaymentSetup();
+        bool CanCheckout();
+        /// Key is the public key that is used for encrypting a native payment's token data
+        void StartCheckout(string key);
```

Comment:
I like that you've named it `StartCheckout` instead of `StartNativeCheckout`. I wonder if calling it just `Start` would be too much? On my PR I've updated `IWebPayable` to be `IWebCheckout` and named the variable in Cart `webCheckout`. This way, the code reads like `webCheckout.Start(url)`. What do you think?